.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 /dist
 /build
 /lib
+.vscode
```

Comment:
Could you remove this? I think local `.gitignores` are a nice way to highlight untracked folders that are generated by scripts in the repo and .vscode seems like a platform specific thing.